src/common/snippets/src/lowered/pass/init_loops.cpp

```diff
@@ -72,23 +72,26 @@ inline void init_is_incremented(LoopPort& port, size_t loop_id) {
     }
 }
 
-inline int64_t get_ptr_increment(const LoopPort& loop_port, size_t work_amount) {
+inline int64_t get_ptr_increment(const LoopPort& loop_port, size_t work_amount, size_t input_port_count, size_t output_port_count) {
```

Comment:
Why do we need to pass both `input_port_count` and `output_port_count`, when only one of them is actually used (L86 and L89)?

Reply:
To set only on parameter `port_count` here, we should set correct `input`/`output` port count in `init_runtime_parameters` on L144. For that we should write the same `if-else` block with exception throwing if `type` is unkown.

However, I agree that `get_ptr_increment` should have only on parameter. Because of that I decided to check for only `input` type on L144 when `init_runtime_parameters` throws exception if `type` is unkown.

It seems like good solution since it's safe anyway. Thanks